abis/FontProject.json

```diff
@@ -1,192 +1,604 @@
+  "linkReferences": {},
+  "deployedLinkReferences": {}
+}
```

Comment:
As you told me before JP this is taken from the compile artifacts generated with hardhat and not from the block explorer, right?